app/frontend/src/features/Contribute.tsx

```diff
@@ -0,0 +1,98 @@
+import { makeStyles, Typography } from "@material-ui/core";
+import { LOGIN } from "features/auth/constants";
+import DesktopAuthBg from "features/auth/resources/desktop-auth-bg.jpg";
+import { Link } from "react-router-dom";
+import { loginRoute } from "routes";
+
+import { COUCHERS } from "../constants";
+import ContributorForm, { CONTRIBUTE, JOIN_THE_TEAM } from "./contributorForm";
+
+const useStyles = makeStyles((theme) => ({
+  bg: {
+    height: "100%",
+    width: "100%",
+    backgroundColor: theme.palette.common.white,
+    // backgroundAttachment: "scroll, local",
+    backgroundPosition: "top center",
+    backgroundRepeat: "no-repeat",
+    backgroundImage: `url(${DesktopAuthBg})`,
+    backgroundSize: "cover",
+    position: "fixed",
+    zIndex: -1000,
+  },
+  page: {
+    boxSizing: "border-box",
+    display: "flex",
+    height: "100vh",
+    alignItems: "flex-start",
+    flexDirection: "column",
+    padding: 0,
+    width: "100%",
+  },
+  content: {
+    flexDirection: "column",
+    height: "100%",
+    margin: "0 auto",
+  },
+  nav: {
+    display: "flex",
+  },
+  link: {
+    borderRadius: theme.shape.borderRadius / 3,
```

Comment:
```suggestion
    borderRadius: theme.shape.borderRadius,
```
I've changed `borderRadius` value in the theme recently if you rebase with `develop` so don't need to divide by 3 anymore